src/native-common/ScrollView.tsx

```diff
@@ -40,9 +40,11 @@ export class ScrollView extends ViewBase<Types.ScrollViewProps, {}> implements R
         // TODO: #737970 Remove special case for UWP when this bug is fixed. The bug
         //   causes you to have to click twice instead of once on some pieces of UI in
         //   order for the UI to acknowledge your interaction.
-        const keyboardShouldPersistTaps = RN.Platform.OS === 'windows' ?
-            true :
-            this.props.keyboardShouldPersistTaps;
+        const keyboardShouldPersistTaps = (
+            RN.Platform.OS === 'windows' ? 'always'
```

Comment:
@steida that's a good suggestion but I don't think prettier supports TypeScript yet (prettier/prettier#13).